webapp/components/Notification/Notification.vue

```diff
@@ -70,14 +56,36 @@ export default {
 </script>
 
 <style lang="scss">
-.notification.read {
-  opacity: $opacity-soft;
-}
-.notifications-card {
-  min-width: 500px;
-}
-.comment-notification-header {
-  font-weight: 700;
-  margin-right: 0.1rem;
+.notification {
+  margin-bottom: $space-base;
+
+  &:first-of-type {
+    margin-top: $space-x-small;
+  }
+
+  &.--read {
+    opacity: 0.5;
```

Comment:
```suggestion
    opacity: $opacity-disabled;
```
avoid magic numbers?